x/concentrated-liquidity/incentives_test.go

```diff
@@ -3615,8 +3672,35 @@ func (s *KeeperTestSuite) TestIncentiveTruncation() {
 	s.Require().True(incentives.IsZero())
 
 	// Incentives should now be claimed due to lack of truncation
-	s.Ctx = s.Ctx.WithBlockTime(s.Ctx.BlockTime().Add(time.Minute * 51))
+	s.Ctx = s.Ctx.WithBlockTime(s.Ctx.BlockTime().Add(time.Hour * 6))
```

Comment:
Why the change of this value?

Reply:
The current tick liquidity and scaling factor value were changed so this threshold changes as well.

The goal is to show that with the increased scaling factor, we do not get a truncation within the expected range of increase